lazyflow/operators/ioOperators/opStreamingHdf5Reader.py

```diff
@@ -126,7 +133,14 @@ def execute(self, slot, subindex, roi, result):
         if result.flags.c_contiguous:
```

Comment:
this check should be done if the shapes match only

Reply:
i didn't understand this point